ServerApp/app/user/controllers.py

```diff
@@ -0,0 +1,73 @@
+from flask import *
+from sqlalchemy.exc import IntegrityError
+from app import *
+from .models import User
+import json
+
+mod_user = Blueprint('user', __name__)
+
+NETWORK_SECRET = 'PASSWD'
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Possible hardcoded password: 'PASSWD'](https://app.codacy.com/manual/rejahrehim/SecureTea-Project/pullRequest?prid=5573783)

Reply:
Intended as a part of server configuration